The exp satisifies all constraints: "Synth Function Passed All Constraints"
(define-fun qm-foo ((x Int)) Int (- x (qm x (- x 1))))
thread 'main' panicked at src/main.rs:54:14:
explicit panic
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
